recipe/meta.yaml

```diff
@@ -35,70 +35,71 @@ source:
       # - patches/amd-roc-hcc-2.7.0.diff  # [variant == "hcc"]
       # ROOT patches to clang 16
       - patches/root/0001-Remove-warning-about-CMake-3.20.0-in-LLVM-17.patch  # [variant and variant.startswith("root_")]
```

Comment:
You could actually put all this into a 
```
{% if variant and variant.startswith("root_") %}
      - patches/root/0001-Remove-warning-about-CMake-3.20.0-in-LLVM-17.patch
      - ...
{% endif %}
```
block, and save yourself all the selectors

Reply:
Will do, thanks!